pkg/vfs/accesslog.go

```diff
@@ -30,7 +32,7 @@ var (
 		Name:    "fuse_ops_durations_histogram_seconds",
 		Help:    "Operations latency distributions.",
 		Buckets: prometheus.ExponentialBuckets(0.0001, 1.5, 30),
-	}, []string{"method"})
+	}, []string{"method", "errno"})
```

Comment:
there will be too many metrics, especially for hisogram

Reply:
Maybe we can have a single metric for all errors.

Reply:
By `too many metrics`, do you mean too many metrics on the client side or tsdb side?

What do you think I move the statistic to a counter, such as:
```suggestion
	opsIOErrors = prometheus.NewCounterVec(prometheus.CounterOpts{
		Name: "fuse_ops_io_errors",
		Help: "Number of IO errors.",
	}, []string{"method", "errno"})
```

Reply:
both.

The combination of method and errno could be hundreds. A single counter or with single label should be enough for monitoring. For further investing, we should collect the accesslog to find out the details.

Reply:
Updated - added a new metric for io errors

Reply:
The new metric has too much labels, please pick one

Reply:
Done